src/components/MainNavBar.js

```diff
@@ -0,0 +1,142 @@
+import React, { useContext } from 'react';
+// import axios from 'axios';
+import { NavLink } from 'react-router-dom';
+import logo from '../img/Yatch-House.png';
+import './pages/mainpage.css';
+
+import { AuthContext } from '../context/AuthContext';
+
+const SideNavbar = () => {
+  const { user, dispatch } = useContext(AuthContext);
+
+  const isAdmin = (user) => {
+    if (user) {
+      return user.data.admin;
+    }
+    return false;
+  };
+
+  const handleLogout = async () => {
+    // try {
+    //   await axios.delete('http://localhost:3001/users/sign_out');
+    // } catch (err) {
+    //   console.log(err);
+    // }
```

Comment:
- [ ] Please, consider deleting this old piece of commented code to improve the cleanliness of the repo. 👍